bootloader/src/pyi_global.c

```diff
@@ -180,12 +239,21 @@ pyi_global_printf(const char *fmt, ...)
 {
     va_list v;
 
-    fprintf(stderr, "[%d] ", getpid());
+    /* Sent 'LOADER text' messages to stderr. */
+#if defined(_WIN32)
+	char utf8_formatted[1024];
```

Comment:
OK, I will use vsnprintf() twice, malloc() and free(). Indentation Issue also I Agree.

Reply:
I would not go for malloc and free since these can fail, too. Using the stack is less error prone.

Just reason (in a commend) why 1024 and not 512 or 2048?

Reply:
Sorry, I can't explain the logical reason why 1024 I used. No source exists the value can rely on.
Do you have any suggestion about another buffer size or the other techniques?

Reply:
No. Just MBTXTLEN. That's because I said not logical.

Reply:
So what about `MBTXTLEN * 2`?